skimage/measure/mc_meta/visual_test.py

```diff
@@ -0,0 +1,72 @@
+""" 

```

Comment:
Getting this working on CI sounds painful (especially with OpenGL in the mix). In principle, the quantitative tests should be sufficient. The purpose of this file is merely to do some quick comparisons of the algorithms and verify output visually during development. I added it to the repo because it might be useful for others (or future me's) when making modifications to the algorithms.